mauleyzaola/maupod
# Maupod 142

Fixes #142

## Test

* Open any album
* Play song
* Resize browser. Spectrum should be consistently resized on refresh
